### ISSUE TITLE
Stuck

### ISSUE THREAD

**Issue by ardimasp** [TaigahG/FSM-Stopwatch#1]
When I tried your simulation and when the current state shows 1 0, I can't escape from there.

**Comment by TaigahG** [comment]
yes sr, I have been looking for the solutions but I still couldn't find it and that's what I have been asking you the solutions. I have tried the polling method you recommended to me but it's still stuck.
